### DISCUSSION TITLE
KeePass 2 yubikey compatible?

### DISCUSSION THREAD

**Discussion by w15eacre** [keepassxreboot/keepassxc#11611] [Q&A] [answered]
I had some problems using a database created by **KeePassXC (Yubikey)** in **KeePass 2** on **Windows**. I installed the plugins **KeeChallenge** and **MultiCipher**, but when I tried to open the database, **KeePass 2** tried to find an XML file to decrypt the kdbx file. As far as I know, it is the XML file that stores some information about Challenge-Response. I store my database in the cloud, and storing the XML file there isn't a good idea, in my opinion. Could you give me some advice about this?

**Comment by droidmonkey** [comment, marked as answer]
We are not compatible with the yubikey plugins. You cannot use yubikey between keepassxc and keepass 2
